osu.Game/Overlays/Comments/DrawableComment.cs

```diff
@@ -257,39 +273,60 @@ protected override void LoadComplete()
                     this.FadeTo(show.NewValue ? 1 : 0);
             }, true);
             childrenExpanded.BindValueChanged(expanded => childCommentsVisibilityContainer.FadeTo(expanded.NewValue ? 1 : 0), true);
+            childComments.BindValueChanged(children =>
+            {
+                children.NewValue.ForEach(c =>
+                {
+                    if (!loadedChildren.Contains(c))
```

Comment:
Possibly nitpicky again, but this `Contains` on a list makes this quadratic. If possible I'd want to see `loadedChildren` being a `Dictionary` or `HashSet`, unless the order of elements matters.

Reply:
Order of the elements matters because of the `Sort` usage. We are receiving already ordered comments